app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+
+    if len(command) == 3:
```

Comment:
Good. Now you can unpack into variables and make the check, as you already did in the previous commit.

Reply:
Ok, but I don't understand why you need to check for length == 3? everything else was done in the last commit